src/components/AddressResolver/AddressesTable.tsx

```diff
@@ -64,6 +66,24 @@ interface AddressesTableProps {
 
 export const AddressesTable = styled(({ className, isNewEndpoint, setNewEndpoint }: AddressesTableProps) => {
   const { thirdPartyAPIEndpoints, removeThirdPartyAPIEndpoint } = useThirdPartyAPIEndpoints();
+  const { showOverlay, setOverlayVisible } = useContext(OverlayContext);
+
+  const deleteAddress = (index: number) => {
+    removeThirdPartyAPIEndpoint(index);
+    setOverlayVisible(false);
+  };
+
+  const onOverlayHandler = (name: string, index: number) => {
```

Comment:
```suggestion
  const onRemoveEndpoint = (name: string, index: number) => {
```

Suggest to say what is does instead.